save-frontend/src/main/kotlin/com/saveourtool/save/frontend/utils/OrganizationMenuBar.kt

```diff
@@ -16,4 +19,22 @@ enum class OrganizationMenuBar(private val title: String? = null) {
      * @return title or name if title is not specified
      */
     fun getTitle() = title ?: name
+
+    companion object : MenuBar<OrganizationMenuBar> {
+        override val defaultTab: OrganizationMenuBar = INFO
+        val listOfStringEnumElements = OrganizationMenuBar.values().map { it.name.lowercase() }
+        override val regex = Regex("/project/[^/]+/[^/]+/[^/]+")
+        override var paths: Pair<String, String> = "" to ""
+        override fun valueOf(elem: String): OrganizationMenuBar = OrganizationMenuBar.valueOf(elem)
+        override fun values(): Array<OrganizationMenuBar> = OrganizationMenuBar.values()
+        override fun findEnumElements(elem: String): OrganizationMenuBar? = values().find { it.name.lowercase() == elem }
+        override fun setPath(shortPath: String, longPath: String) {
+            paths = shortPath to longPath
+        }
+
+        override fun returnStringOneOfElements(elem: OrganizationMenuBar): String = elem.name
+
+        override fun isNotAvailableWithThisRole(role: Role, elem: OrganizationMenuBar?, flag: Boolean?): Boolean = ((elem == SETTINGS) && !role.isHigherOrEqualThan(Role.ADMIN)) ||
+                ((elem == CONTESTS) && (!role.isHigherOrEqualThan(Role.OWNER) || flag == false))
```

Comment:
And again you use `!role.isHigherOrEqualThan(Role)` instead of  `role.isLowerThan(Role)`

Reply:
Done